documentation/API-design-guidelines.md

```diff
@@ -1068,20 +1068,55 @@ Below considerations should be checked when an API is documented:
 ### 11.1 General Information
 
 This part must include the following information:
-- API Version in the next format: X.Y.Z.
 - API title with public name.
 - A brief description of the main functions of the API.
+- API Version in the format defined in [Chapter 5. Versioning](#5-versioning)
 - API Terms of Service.
 - Contact information, with name, email and website of the API Holder.
 - Licence information (name, website…)
-- Schemes supported (HTTP, HTTPS…)
-- Allowed response formats (“application/json”, “text/xml”…)
-- Response format (“application/jwt”…)
+- API server and base URL
 - Global `tags` object if tags are used for API operations
   
-<p align="center">
-<img src="./images/guidelines-fig-14.png" width="400"/>
-</p>
+The `info` object shall have the following content:
+
+```yaml
+info:
+  # title without "API" in it, e.g. "Number Verification"
+  title: Number Verification
+  # description explaining the API, part of the API documentation 
+  # text explaining how to fill the "Authorization and authentication" - see section 11.6
+  description: |
+    This API allows to verify that the provided mobile phone number is the one used in the device. It
+    verifies that the user is using a device with the same mobile phone number as it is declared.
+    ### Authorization and authentication
+    CAMARA guidelines defines a set of authorization flows ...
+  # API version - Aligned to SemVer 2.0 according to CAMARA versioning guidelines
+  version: 1.0.1
+  # Link to the page that describes the terms of service - to be replaced by the provider's terms
+  termsOfService: http://example.com/terms/
+  # Contact information: name, email, URL
+  contact:
+    name: API Support
```

Comment:
One doubt 
which values are expected to be used in the APIs (I assume we will have to insert this information in every API so i think we could have some alignment regarding the value to be indicated to use the same approach among APIs or it is up to the WG the value to be provided).
Anycase, all of them (name, email, url) has to be indicated within the yaml, doesn`t?


name: <WG name> support -> e.g.  Blockchain Public Address  Support
email: mail distribution list of the WG -> e.g.  sp-bpa@lists.camaraproject.org
url: link to the WG site in CAMARA gtihub -> e.g. https://github.com/camaraproject/BlockchainPublicAddress

Reply:
I think it is assumed that the contact information will be changed by each API Provider (Operator or Aggregator) when publishing API specification as the documentation for their service - possibly it should be explained in API Design Guidelines here in chapter 11.
When yaml is hosted in CAMARA Github all contact info is in main README.md
@tanjadegroot @hdamker WDYT?

Reply:
I think we should include the information that is relevant for CAMARA in our specs, as each API provider will adapt our OAS spec to whatever template or format they use in their portals. `title` and  `version` are the only required sub-sections for `info`  section. If we decided to add the `contact` sub-section, I would fill it with contact details for CAMARA.

Reply:
I had added such contact subsection with Camara relevant details in the [common.yaml ](https://github.com/camaraproject/Commonalities/blob/main/artifacts/CAMARA_common.yaml )as a part of the last release and would personally prefer this format as well.

<img width="248" alt="image" src="https://github.com/camaraproject/Commonalities/assets/77152136/f9e7b062-d10b-4785-b295-f9a48194b067">

Reply:
Following the discussion in Release Management: https://wiki.camaraproject.org/display/CAM/2024-06-04+Release+WG+Minutes and https://github.com/camaraproject/Commonalities/pull/214#discussion_r1626072362 I have removed `contact` and `termsOfService` from the info object and added some explanation.

See the changes: https://github.com/camaraproject/Commonalities/pull/214/commits/2421afb69f4d59597a0aa10f6022e9387cc3483d.